Clients/Xamarin.Interactive.Client/ViewInspector/DelegateCommand.cs

```diff
@@ -8,7 +8,7 @@
 using System;
 using System.Windows.Input;
 
-namespace Xamarin.Interactive.Client.Windows.Commands
+namespace Xamarin.Interactive.Client.ViewInspector
```

Comment:
Why change this namespace/move this file?

Reply:
Because it is in the shared model now?

Reply:
Still feels weird—I would’ve put it in its own Commands namespace maybe, but it’s neither here nor there really.